### PR TITLE
Add avatar support for Open Hub & OpenSUSE

### LINKED ISSUE #62: Add avatar for openSUSE
Relavent files to make changes to:
- websites.yml
- username_api.py
- avatar testing

### PR DESCRIPTION
**Pull request by pavi2410**
SUMMARY: Added avatar support for Openhub & openSUSE by parsing their respective web pages.

I have fixed both issues, i.e, #62 & #63.

### CODE REVIEW

**Comment by jayvdb** [comment]
Please do not close Pull Requests.  You need to learn how to use git.  In the future, ask our mentors for help on Zulip.

**Comment by jayvdb** [comment]
This was a continuation of https://github.com/manu-chroma/username-availability-checker/pull/72

**Comment by jayvdb** [comment]
Please fix your commit message to comply with http://coala.io/commit

File: .gitignore [hunk 1]
```diff
@@ -14,3 +14,5 @@ __pycache__/
 .env
 
 static/logs.log
+
+\.idea/
```
Comment (jayvdb) [inline .gitignore]: this doesnt belong as part of this PR.

Comment (pavi2410) [reply .gitignore]: So should I create a separate PR?

Comment (pavi2410) [reply .gitignore]: I can do that in 3 hours

**Comment by pavi2410** [comment]
I hope you will like the updated PR description

File: .gitignore (hunk 1 above)
Comment (manu-chroma) [reply .gitignore]: > So should I create a separate PR?

Noo. Please amend your commit to add which issue your PR closed. See: http://api.coala.io/en/latest/Developers/Git_Basics.html#follow-up

During amend you can remove the changes you made in ``.gitignore``

**Comment by manu-chroma** [comment]
can you squash your commits?

**Comment by pavi2410** [comment]
You can do squash and commit from Github, right?

File: username_api.py [hunk 2]
```diff
@@ -46,6 +46,11 @@ def get_avatar(website, username):
         if not result or not result[0]:
             return None
         result = result[0]
+    elif 'html_selector' in data:
+        soup = BeautifulSoup(response.text, 'html.parser')
+        img = soup.select(data['html_selector'])
```
Comment (jayvdb) [inline username_api.py]: This is a list.  The variable should end with a `s`.  e.g. `images` or `selected_items`.

**Comment by pavi2410** [comment]
Finally. 😄

**Pull request merged**